cogboard-webapp/src/components/SnackbarVariantContent/styled.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import styled from "@emotion/styled/macro";
+import {SnackbarContent} from "@material-ui/core";
+
+import {mapVariantToColor} from "./helpers";
+
+export const SpanIcon = styled.span`
+    display: flex;
+    alignItems: center;
+    justifyContent: center;
+`;
+
+export const IconStyles = {
```

Comment:
Styled... Maybe it is the highest time to make an IconVariant component with passed icon map object displaying icon based on other props. We have already StatusIcon component mad by @marcin-witaszek. It is good example, so check this out and unify/refactor if possible.